### DISCUSSION TITLE
error SEB

### DISCUSSION THREAD

**Discussion by coubalovajana** [SafeExamBrowser/seb-win-refactoring#94] [Q&A]
Hi, I installed SEB few weeks ago, and when I wanna open trying key from the exam, it shows me this errors. Can you please help me? Thank you.
![problém seb](https://user-images.githubusercontent.com/85411485/120901632-6fbe1e00-c63c-11eb-9d12-54f205d8261b.png)
![seb problém 2](https://user-images.githubusercontent.com/85411485/120901646-806e9400-c63c-11eb-8ead-02d0f70ca668.png)

**Comment by dbuechel** [comment]
There appears to be an issue with the configuration file you're trying to open. Could you please provide the respective runtime log file, to be found under `%LocalAppData%\SafeExamBrowser\Logs`?
